src/models/BaseModel.py

```diff
@@ -80,39 +78,45 @@ def preprocess_data(self):
         print('Pre-processing begins')
         # Description: consisting of 5 features existence of words below in the description:
         # 'excellent', 'graduate', 'immediate', 'junior', 'urgent'
-        onehot_encoded_desc_words = get_one_hot_encoded_words(self.description_feature)
+        (onehot_encoded_desc_words, col_names_1) = \
```

Comment:
don't change it if i'm the only one bothered by it, but shouldn't the variables be calls col_names_contract, col_names_desc and so on?